deployment/single-server/app.yml

```diff
@@ -0,0 +1,106 @@
+---
+
+# Invoke using:
+#
+#     ansible-playbook app.yml --become --ask-become-pass
+#
+
+- name: Catalogi Importer
+  hosts: all
+
+  collections:
+    - maykinmedia.commonground
+
+  vars:
+    # Application
+    django_app_docker_name_prefix: objecttypes
+    django_app_docker_domain: TODO
+    django_app_docker_secret_key: 'TODO'
+    django_app_docker_package_name: objecttypes
+    django_app_docker_https: false
+
+    ## Application database settings
+    django_app_docker_db_host: TODO
+    django_app_docker_db_name: objecttypes
+    django_app_docker_db_username: objecttypes
+    django_app_docker_db_password: TODO
+
+    ## Application docker container
+    django_app_docker_version: latest
+    django_app_docker_image_name: maykinmedia/objecttypes-api
+    django_app_docker_replicas: 1
+    django_app_docker_port_start: 12000
+
+    django_app_docker_volumes:
+      - name: "{{ django_app_docker_name_prefix }}-media"
+        mount: "/app/media"
+        mode: "rw"
+        owner: 1000
+      - name: "{{ django_app_docker_name_prefix }}-private-media"
```

Comment:
can remove this whol volume block i think

Reply:
Sure, I'll remove the whole `django_app_docker_volumes` block